pipelines/broad/arrays/single_sample/Arrays.documentation.md

```diff
@@ -11,12 +11,19 @@ The Arrays Pipeline takes the inputs described below. Inputs are grouped by func
 
 
 *   **Sample-specific**
-    *   chip_well_barcode. The unique identifier of the section of the chip on which the sample was run on. This is the concatenation of the chip barcode and the coordinates of the sample on the chip. Example: 7991775143_R01C01
-    *   analysis_version. A numeric value that is used to track the number of times the pipeline has been run on this chip_well_barcode.
+    *   chip_well_barcode. The unique identifier of the section of the chip on which the sample was run on. This is the concatenation of the chip barcode and the coordinates of the sample on the chip. 
+        * Example: 7991775143_R01C01
+
+    *   analysis_version_number. A numeric value that is used to track the number of times the pipeline has been run on this chip_well_barcode.
```

Comment:
it looks like this input is optional in the pipeline, should that be indicated here?

Reply:
Yes.  Good catch!